src/staging/src/pages/404.js

```diff
@@ -1,46 +1,42 @@
 import * as React from "react"
-import { Link } from "gatsby"
 
-import BetaBanner from "../components/betaBanner"
+import Header from "../components/header"
```

Comment:
On my local I'm getting a blank blue banner where the nav usually is, and on mobile, the hamburger menu is there, and opens a blank panel.

Reply:
Good catch! I totally missed including the menu data...Thank you 🙏